libraries/chain/include/eosio/chain/fork_database.hpp

```diff
@@ -11,6 +11,7 @@ namespace eosio::chain {
    using block_branch_t = std::vector<signed_block_ptr>;
    enum class mark_valid_t { no, yes };
    enum class ignore_duplicate_t { no, yes };
+   enum class check_root_t { no, yes };
```

Comment:
Minor thing, but seems like this should be called `include_root_t`

Reply:
That's actually the named I used first, but it felt wrong for some reason and I prefer `check_root_t`. If you want to change it later I'm fine with it.